webapp/components/compat-2021.js

```diff
@@ -254,3 +255,228 @@ class Compat2021Summary extends PolymerElement {
 }
 window.customElements.define(Compat2021Summary.is, Compat2021Summary);
 
+class Compat2021FeatureChart extends PolymerElement {
+  static get template() {
+    return html`
+      <style>
+        .chart {
+          /* Reserve vertical space to avoid layout pop. Should be kept in sync with
+             the JavaScript defined height. */
+          height: 350px;
+          margin: 0 auto;
+        }
+      </style>
+
+      <h3>Individual Features</h3>
+
+      <!-- TODO: replace with paper-dropdown-menu -->
+      <div>
+        <label for="featureSelect">Focus area:</label>
+        <select id="featureSelect">
```

Comment:
A problem here is there's no way to go back to the overall view once you've chosen a subview. Can you add an explicitly "everything" option which is selected by default?

Reply:
I'm not clear what the purpose of that would be? Currently, as requested by Chris, the overall view is just blank (https://ecosystem-infra.github.io/wpt-results-analysis/compat).

However typing that, it becomes instantly clear that the no-feature view should probably be a graph of the overall score over time -_-. Does that SGTY? If so, it will take time to put it together so I'd suggest we keep this behavior for now.

Reply:
Aha, so I had imagined that the overall big numbers were also updating, and that there's no way to go back.

An overall graph would perhaps be nice, but yeah let's leave this for now.